.github/workflows/test-package.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  # Run on PRs and pushes to the default branch.
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 0 * * 0"
+
+env:
+  PUB_ENVIRONMENT: bot.github
```

Comment:
Should we be looking at automating this in pub? We can't rely on external folks using github actions to configure this.

Reply:
Yes!

I tried to get something in the pub client – but I got pushback - https://github.com/dart-lang/pub/pull/2759